src/view/creating-form-view.js

```diff
@@ -0,0 +1,183 @@
+import { createElement } from '../render';
+
+const createAddingFormTemplate = () => (`
+<li class="trip-events__item">
+<form class="event event--edit" action="#" method="post">
+<header class="event__header">
```

Comment:
Понимаю что в шаблонных строках не совсем удобно но давай соблюдать форматирование разметки. Здесь и далее во всех компонентах
```suggestion
const createAddingFormTemplate = () => (`
  <li class="trip-events__item">
    <form class="event event--edit" action="#" method="post">
      <header class="event__header">
```